src/main/resources/assets/simplemagnets/lang/pt_br.json

```diff
@@ -12,13 +16,21 @@
   "gui.advancedmagnet.items.increase": "Aumentar Alcance",
   "gui.advancedmagnet.items.decrease": "Diminuir Alcance",
   "gui.advancedmagnet.items.range": "Alcance ($number$x$number$x$number$ Blocos)",
-  "gui.advancedmagnet.xp.on": "Desligado",
-  "gui.advancedmagnet.xp.off": "Off",
+  "gui.advancedmagnet.xp.on": "Ligado",
+  "gui.advancedmagnet.xp.off": "Desligado",
   "gui.advancedmagnet.xp.increase": "Aumentar Alcance",
   "gui.advancedmagnet.xp.decrease": "Diminuir Alcance",
   "gui.advancedmagnet.xp.range": "Alcance ($number$x$number$x$number$ Blocos)",
-  "gui.advancedmagnet.whitelist.on": "Lista Branca",
-  "gui.advancedmagnet.whitelist.off": "Lista Negra",
+  "gui.advancedmagnet.whitelist.on": "Ignorar",
+  "gui.advancedmagnet.whitelist.off": "Pegar",
+  "gui.simplemagnets.demagnetization_coil.durability.on": "Combinar NBT",
+  "gui.simplemagnets.demagnetization_coil.durability.off": "Ignorar NBT",
+  "gui.simplemagnets.demagnetization_coil.range.increase": "Aumentar Alcance",
+  "gui.simplemagnets.demagnetization_coil.range.decrease": "Diminuir Alcance",
+  "gui.simplemagnets.demagnetization_coil.range": "Alcance ($number$x$number$x$number$ Blocos)",
   "keys.category.simplemagnets": "Simple Magnets",
   "keys.simplemagnets.toggle": "Alternar ímã"
```

Comment:
I think there's a missing comma here. Kinda hard to see on my phone though.

Reply:
Done